### PR TITLE
Max images, nullability/generics, button color, sample app launch screen

### PR DESCRIPTION
**Pull request by designatednerd**

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal/ios-developers (and @lquigley since this is open source!), anyone wanna take a look at some fun stuff TOMORROW?

**Comment by designatednerd** [comment]
@vokal/ios-developers Anyone? Bueller? Bueller?

**Comment by bryanluby** [comment]
On it

File: Pod/Classes/Categories/UICollectionView+VOK.h [hunk 1]
```diff
@@ -17,6 +19,8 @@
  *
  *  @return An array of NSIndexPaths of elements in the specified rect.
  */
-- (NSArray *)vok_indexPathsForElementsInRect:(CGRect)rect;
+- (NSArray<NSIndexPath *> *__nullable)vok_indexPathsForElementsInRect:(CGRect)rect;
```
Comment (bryanluby) [inline Pod/Classes/Categories/UICollectionView+VOK.h]: Why not `(nullable NSArray<NSIndexPath *> *)...`?

File: VOKMultiImagePicker-iOS.podspec [hunk 2]
```diff
@@ -17,7 +8,7 @@ Pod::Spec.new do |s|
   s.author           = { "Luke" => "luke.quigley@vokalinteractive.com" }
```
Comment (lquigley) [inline VOKMultiImagePicker-iOS.podspec]: This email address doesn't exist anymore. :disappointed:

Comment (designatednerd) [inline VOKMultiImagePicker-iOS.podspec]: :cry: :shower:  - I'll point it at one of our more generic ones. Miss ya!

**Comment by designatednerd** [comment]
@bryanluby @lquigley Changes pushed

**Comment by bryanluby** [comment]
Good to pull

**Pull request merged**